src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/MicrosoftApiDesignGuidelinesAnalyzersResources.resx

```diff
@@ -1004,4 +1004,28 @@
     <value>{0} should define operator(s) '{1}' and Equals since it implements IComparable.</value>
     <comment>1 is a comma-separated list</comment>
   </data>
+  <data name="StringBuilderShouldUseSubstringOverloadWithOneParameterTitle" xml:space="preserve">
+    <value>A</value>
```

Comment:
❗️ We don't necessarily need final strings before merge, but we do need ones that make some bit of sense 😆

Reply:
Of course ;) Didn't yet figure out what the strings are used for, so I left them out. Going to resolve that in the next hours...

Reply:
provided slightly more meaning now, could be parametrized and probably better wordings, but should work for now.